tools/lxdclient/config.go

```diff
@@ -60,13 +60,24 @@ func (cfg Config) UsingTCPRemote() (Config, error) {
 		return cfg, nil
 	}
 
+	/* Here, we rely on the fact that Connect() does a ServerStatus.
```

Comment:
On Tue, Apr 19, 2016 at 03:49:03AM -0700, John Arbash Meinel wrote:
> > @@ -60,13 +60,24 @@ func (cfg Config) UsingTCPRemote() (Config, error) {
> >  		return cfg, nil
> >  	}
> >  
> > +	/* Here, we rely on the fact that Connect() does a ServerStatus.
> 
> Connect() doesn't do a ServerStatus anymore for HTTP connections. Shouldn't you use Finger() if you really want it to trigger it?

Hmm. I could have sworn I read it yesterday and it does, but looking
at it again it seems it doesn't. I'm not sure how this worked given
that it doesn't, but yes, we probably do. Let me re-test and I'll push
a fix.